orchestra/model/configuration/configuration.py

```diff
@@ -315,7 +316,10 @@ def _expand_variables(self, string):
 
 def locate_orchestra_dotdir(cwd=None):
     if cwd is None:
-        cwd = os.getcwd()
+        if globals.orchestra_dotdir is not None:
+            cwd = globals.orchestra_dotdir
+        else:
+            cwd = os.getcwd()
```

Comment:
What semantic do we want to give to `--orchestra-dotdir`?
Should the user pass:
1. `$ORCHESTRA_ROOT/`,
2. `$ORCHESTRA_ROOT/.orchestra`, or
3. any subdir of `$ORCHESTRA_ROOT/` and then autodiscover

`git --git-dir` semantic seems to be closer 2), while AFAIU you're implementing 3).

Reply:
`$ORCHESTRA_ROOT` refers to `<orchestra_dir>/root`, anyway it's number 3, yes it's different than git, but I argue it's more convenient, especially because of the possibility of specifying just `<orchestra_dir>` instead of `<orchestra_dir>/.orchestra`